tests/common.py

```diff
@@ -0,0 +1,24 @@
+from platformdirs.macos import MacOS
```

Comment:
Use fixtures to share argument, not import. Remove the ``tests/__init__.py`` it's not needed.